src/GitHub/Request.hs

```diff
@@ -256,8 +256,8 @@ makeHttpRequest auth r = case r of
     setAuthRequest _                                  = id
 
     getOAuthHeader :: Auth -> RequestHeaders
-    getOAuthHeader (OAuth token) = [("Authorization", BS8.pack ("token " ++ token))]
-    getOAuthHeader _                   = []
+    getOAuthHeader (OAuth token) = [("Authorization", "token " <> token)]
+    getOAuthHeader _             = []
```

Comment:
@zudov good spot, I have no Enterprise installation to test against, so I just leave this as it is.